enos/ansible/enos.yml

```diff
@@ -32,9 +32,8 @@
         tags: ['grafana'],
         when: enable_monitoring | bool }
 
-- name: Rally tasks
+- name: "{{ bench.type }} tasks"
   hosts: disco/bench
   roles:
-    - { role: rally,
-        tags: ['rally'] }
-        
+    - { role: "{{ bench.type }}",
```

Comment:
`bench.type` isn't defined when calling `enos up --pull` (which ony pulls all the needed containers). 
Should we define a default `bench` variable somewhere ?

Reply:
Another solution is ensuring that `--pull` always comes with `--workloads` so you know the `bench.type`. Something like:

```
enos bench [--workload=WORKLOAD [--pull]]
```

Reply:
When I rewrote the playbook the desired behaviour was the following: 
- using `pull` must pull all possible containers (rally + shaker)

but we could relax that if that's required.
Passing `workload` may solve the problem in this direction but this will require to parse the workload file in the up phase. (or we defer the pulling to the bench phase by adding a `pull` option)